src/aioswitcher/bridge.py

```diff
@@ -259,7 +259,12 @@ def _parse_device_from_datagram(
                     parser.get_thermostat_remote_id(),
                 )
             )
-        elif device_type and device_type.category == DeviceCategory.LIGHT:
+
+        elif (
+            device_type
+            and device_type.category == DeviceCategory.LIGHT
+            and device_type in (DeviceType.LIGHT_SL01, DeviceType.LIGHT_SL01_MINI)
```

Comment:
I am not sure if we should start differentiating between mini and regular versions (we didn't do it for the V4 & V4 mini) since they have identical protocol and the only difference is mechanical size

Reply:
We did do that for the Runner and Runner Mini..

Reply:
Actually we so seperete V4 ans V4 mini it just called Switcher Mini